addons/purchase/models/res_partner.py

```diff
@@ -52,6 +53,15 @@ def _compute_supplier_invoice_count(self):
     def _commercial_fields(self):
         return super(res_partner, self)._commercial_fields()
 
+    def open_purchase_matching(self):
+        return {
+            'type': 'ir.actions.act_window',
+            'name': _("Purchase Matching"),
+            'res_model': 'purchase.bill.line.match',
+            'domain': [('partner_id', '=', self.id), ('company_id', 'in', [self.env.company.id])],
```

Comment:
@chklop was ok with this...
However, I think you're right. we should be able to continue the flow in the company of the current bill/PO...
but since we don't have time/ po's to test it, let's keep it this way and we can improve later